addons/payment_buckaroo/controllers/main.py

```diff
@@ -17,15 +18,13 @@ class BuckarooController(http.Controller):
     _reject_url = '/payment/buckaroo/reject'
 
     @http.route([
-        '/payment/buckaroo/return',
-        '/payment/buckaroo/cancel',
-        '/payment/buckaroo/error',
-        '/payment/buckaroo/reject',
+        _return_url,
+        _cancel_url,
+        _exception_url,
+        _reject_url,
     ], type='http', auth='public', csrf=False)
```

Comment:
```suggestion
    ], type='http', auth='public', methods=['POST'], csrf=False)
```
Better safe than sorry.